### PR TITLE
Content update: on-demand mention and security question

### PR DESCRIPTION
**Pull request by marcelomazza**
Add on-demand mention to error quota docs and clarify "Is data encrypted in transit?" question.

### CODE REVIEW

File: jekyll/_docs/airbrake-faq/monthly-error-quota.md [hunk 1]
```diff
@@ -12,20 +12,17 @@ quota is refilled at the beginning of each new billing period.
...
-errors until you [upgrade your plan](https://airbrake.io/account/plan/edit) or the
-next billing cycle starts. Upgrading your plan will increase your current
-monthly error quota and errors will be accepted again immediately.
+### Can I control error volume to avoid on-demand charges?
+Yes, you can set a "usage cap" for your account and for individual projects.
+This setting will prevent on-demand charges, but will also mean that if you
+exceed your account's error quota, no more errors will be accepted until your
+next billing cycle begins. You will receive alerts if you get close to exceeding
+your monthly quota. At any point in time you can adjust the usage caps for your
+account or check your current usage via your Account Dashboard.
```
Comment (thompiler) [inline jekyll/_docs/airbrake-faq/monthly-error-quota.md:26]: Users can change their usage caps from the account settings page:
https://airbrake.io/account/edit#tab-usage-caps

Not the account dashboard right?

File: jekyll/_docs/airbrake-faq/monthly-error-quota.md [hunk 2]
```diff
@@ -12,20 +12,17 @@ quota is refilled at the beginning of each new billing period.
...
+If you exceed your account's error quota, additional errors will be charged an
+"on-demand" rate based on your selected plan.
 
-If you completely deplete your monthly error quota, Airbrake will stop accepting
-errors until you [upgrade your plan](https://airbrake.io/account/plan/edit) or the
-next billing cycle starts. Upgrading your plan will increase your current
-monthly error quota and errors will be accepted again immediately.
+### Can I control error volume to avoid on-demand charges?
+Yes, you can set a "usage cap" for your account and for individual projects.
+This setting will prevent on-demand charges, but will also mean that if you
```
Comment (thompiler) [inline jekyll/_docs/airbrake-faq/monthly-error-quota.md:22]: This is not necessarily true depending on the cap they set right? They can set the cap to higher than their base quota if they want to allow for some on-demand usage but not go over a specified amount.

We could say something like:
>This setting can prevent on-demand charges if you set your account cap to your total base quota.

File: jekyll/_docs/airbrake-faq/monthly-error-quota.md (hunk 1 above)
Comment (marcelomazza) [reply jekyll/_docs/airbrake-faq/monthly-error-quota.md:26]: @thompiler thanks, I took that text from the FAQs; I'm updating it here and passing the update for the FAQs too.

**Review by thompiler — APPROVED** [review]
👍

**Pull request merged**